From f5eaedf48ab532218c70c9e0e58c2609a42f3326 Mon Sep 17 00:00:00 2001
From: Eli Collins <eli@apache.org>
Date: Sun, 8 Jul 2012 19:53:14 +0000
Subject: [PATCH 0395/1357] HDFS-3614. Revert unused MiniDFSCluster constructor from HDFS-3049. Contributed by Arun Murthy

git-svn-id: https://svn.apache.org/repos/asf/hadoop/common/trunk@1358825 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit 9ec1c06cbb683f0bce0dbc23ccd4511bab2663e2)
---
 .../org/apache/hadoop/hdfs/MiniDFSCluster.java     |   13 ++++++-------
 1 files changed, 6 insertions(+), 7 deletions(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/MiniDFSCluster.java b/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/MiniDFSCluster.java
index 4152371..7848346 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/MiniDFSCluster.java
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/MiniDFSCluster.java
@@ -416,7 +416,7 @@ public class MiniDFSCluster {
   public MiniDFSCluster(Configuration conf,
                         int numDataNodes,
                         StartupOption nameNodeOperation) throws IOException {
-    this(0, conf, numDataNodes, false, false, false, false,  nameNodeOperation, 
+    this(0, conf, numDataNodes, false, false, false,  nameNodeOperation, 
           null, null, null);
   }
   
@@ -438,7 +438,7 @@ public class MiniDFSCluster {
                         int numDataNodes,
                         boolean format,
                         String[] racks) throws IOException {
-    this(0, conf, numDataNodes, format, true, true, true, null,
+    this(0, conf, numDataNodes, format, true, true, null,
         racks, null, null);
   }
   
@@ -461,7 +461,7 @@ public class MiniDFSCluster {
                         int numDataNodes,
                         boolean format,
                         String[] racks, String[] hosts) throws IOException {
-    this(0, conf, numDataNodes, format, true, true, true, null,
+    this(0, conf, numDataNodes, format, true, true, null,
         racks, hosts, null);
   }
   
@@ -496,7 +496,7 @@ public class MiniDFSCluster {
                         StartupOption operation,
                         String[] racks) throws IOException {
     this(nameNodePort, conf, numDataNodes, format, manageDfsDirs,
-        manageDfsDirs, manageDfsDirs, operation, racks, null, null);
+        manageDfsDirs, operation, racks, null, null);
   }
 
   /**
@@ -529,7 +529,7 @@ public class MiniDFSCluster {
                         StartupOption operation,
                         String[] racks,
                         long[] simulatedCapacities) throws IOException {
-    this(nameNodePort, conf, numDataNodes, format, manageDfsDirs, manageDfsDirs,
+    this(nameNodePort, conf, numDataNodes, format, manageDfsDirs, 
         manageDfsDirs, operation, racks, null, simulatedCapacities);
   }
   
@@ -564,14 +564,13 @@ public class MiniDFSCluster {
                         int numDataNodes,
                         boolean format,
                         boolean manageNameDfsDirs,
-                        boolean enableManagedDfsDirsRedundancy,
                         boolean manageDataDfsDirs,
                         StartupOption operation,
                         String[] racks, String hosts[],
                         long[] simulatedCapacities) throws IOException {
     this.nameNodes = new NameNodeInfo[1]; // Single namenode in the cluster
     initMiniDFSCluster(conf, numDataNodes, format,
-        manageNameDfsDirs, true, enableManagedDfsDirsRedundancy, manageDataDfsDirs,
+        manageNameDfsDirs, true, manageDataDfsDirs, manageDataDfsDirs,
         operation, racks, hosts,
         simulatedCapacities, null, true, false,
         MiniDFSNNTopology.simpleSingleNN(nameNodePort, 0), true, false);
-- 
1.7.0.4

